### PR TITLE
Rename render functions + README improvements

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

**Comment by wazolab** [comment]
@frodrigo If you want to add your helper display function in `./src/utils/helpers.ts`, you could name it following this convention `unfoldedClusterRender...`.
Then you can add this option in README after line 120

File: README.md [hunk 1]
```diff
@@ -53,7 +89,7 @@ map.on('load', () => {
 })
 
 // Create whatever HTML element you want as Cluster
-const clusterMode = (element: HTMLDivElement, props: MapGeoJSONFeature['properties'], size?: number): void => {}
+const clusterRender = (element: HTMLDivElement, props: MapGeoJSONFeature['properties'], size?: number): void => {}
```
Comment (frodrigo) [inline README.md:92]: clusterRenderFn ?

Comment (wazolab) [reply README.md:92]: it's the function itself here, not the option. I think we don't need to specify the `fn` statement in it's name as we know already it's a function. It make sense on the option name as it indicates we are expecting a Function to be passed to this option.
What do you think ?

Comment (frodrigo) [reply README.md:92]: Right

**Pull request merged**